### DISCUSSION TITLE
Roadmap content

### DISCUSSION THREAD

**Discussion by vinzvinci** [SurPathHub/starter-kit#3] [Ideas]
### Create markdown per course in <a href="https://github.com/SurPathHub/starter-kit/tree/main/roadmap">Roadmap</a> folder:
Ex. `web-development.md`, `data-science.md`, `dev-ops.md` and etc.
~~-~~

### Roadmap content
- Short Description of the course.
- `Blog/Vlog link to motivate student (optional)`
- Crash-course link

~~-~~
`The Crash Course should be updated,` `(I prefer 2021)`, `please provide quality content`
#### Target list:
- Web Development Crash Course (please use <a href="https://www.youtube.com/watch?v=VfGW0Qiy2I0">Traversy Media</a> video)
- Data Science Crash Course
- DevOps Crash Course
- Etc.

Language: English

~~-~~
Guide: Learn how to embed the thumbnail of the YouTube video
Link: <a href="https://stackoverflow.com/questions/11804820/how-can-i-embed-a-youtube-video-on-github-wiki-pages">https://stackoverflow.com/questions/11804820/how-can-i-embed-a-youtube-video-on-github-wiki-pages</a>
`
Sample: 
[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](https://www.youtube.com/watch?v=YOUTUBE_VIDEO_ID_HERE)
`
Click <a href="https://github.com/SurPathHub/starter-kit/blob/main/roadmap/web-development.md">web-development.md</a> for example of the content.

~~-~~
<a href="https://github.com/SurPathHub/starter-kit">Starter Kit</a> repository.

Please comment on your suggestion below.

**Comment by srisatya12** [comment]
@vinzvinci I want to share one of the link which is for mobile development 
https://whimsical.com/mobile-development-roadmap-devsoc-WQEsHr372VrnvxBSaggPnc

**Reply by vinzvinci** [reply]
This content is too hard to understand for beginners. 

I suggest make it like this 
https://github.com/SurPathHub/starter-kit/blob/main/roadmap/web-development.md

**Comment by vinzvinci** [comment]
I have already done the web-development.md

**Comment by srisatya12** [comment]
@vinzvinci I'm sharing a playlist for Android App Development : https://www.youtube.com/watch?v=f1UbSm_VxhA&list=PLYxzS__5yYQl31tSVpCDsI6Y2J_YjHhE8

**Reply by vinzvinci** [reply]
I was thinking about Android App Development, Flutter, IOS Mobile development crash course. I don't know much about mobile app, is kotlin is related to android app development?

**Reply by vinzvinci** [reply]
If you find one crash course tackles that 3, that will be good. If not, it's okay to provide each course crash course. Please do comment here for your suggestions.

**Reply by srisatya12** [reply]
> I was thinking about Android App Development, Flutter, IOS Mobile development crash course. I don't know much about mobile app, is kotlin is related to android app development?

@vinzvinci Yes kotlin is used in android app development. Kotlin is more concise than Java because it solves problems within fewer lines of code.

**Reply by srisatya12** [reply]
> I was thinking about Android App Development, Flutter, IOS Mobile development crash course. I don't know much about mobile app, is kotlin is related to android app development?

so you are thinking to get these in the one video or what?

**Reply by srisatya12** [reply]
> If you find one crash course tackles that 3, that will be good. If not, it's okay to provide each course crash course. Please do comment here for your suggestions.

@vinzvinci Yeah that's a great idea I'll see into it.

**Reply by vinzvinci** [reply]
Yes I like to have it on one video introducing that 3, but I don't know if it's available on YouTube or to other platform.

**Reply by vinzvinci** [reply]
Second option: Separate that 3

**Reply by srisatya12** [reply]
> Yes I like to have it on one video introducing that 3, but I don't know if it's available on YouTube or to other platform.

@vinzvinci There's a crash course for flutter and IOS dev. But I haven't found for those three.
https://www.youtube.com/watch?v=GLSG_Wh_YWc

**Reply by srisatya12** [reply]
@vinzvinci did you see the crash course I have given the link.

**Reply by vinzvinci** [reply]
Hey @srisatya1197, I found one. I think this much better for the roadmap: <a href="https://www.youtube.com/watch?v=WvwwL0TwH6U">How to Become a Mobile Developer in 2021</a>

Feel free to add this on <a href="https://github.com/SurPathHub/starter-kit/blob/main/roadmap/mobile-development.md">mobile-development.md</a>

Commit title: **Add mobile-development content**

**Reply by vinzvinci** [reply]
Hi @srisatya1197, I'll start to put it there now coz my peers are starting to look at it now.

**Reply by srisatya12** [reply]
@vinzvinci That completely fine.

**Comment by vinzvinci** [comment]
Finally: https://github.com/SurPathHub/starter-kit/tree/main/roadmap

**Reply by srisatya12** [reply]
@vinzvinci Hmm that looks good.

**Comment by srisatya12** [comment]
@vinzvinci Can we still add some roadmaps?

**Reply by vinzvinci** [reply]
Yes
